### ISSUE TITLE
remove left icon

### ISSUE THREAD

**Issue by superandrew** [galio-org/galio#97]
**Is your feature request related to a problem? Please describe.**
I would like to remove the left icon from the navbar. It doesn't seems possible

**Describe the solution you'd like**
One possible solution would be to provide a leftIcon nullable property

**Describe alternatives you've considered**
To color the icon the same color of background 🙇‍♂️

**Comment by palingheorghe** [comment]
Hi @superandrew ! Yes, that's a problem. It should really be easier to remove the left Icon completely. Do you have an idea about how would you go about resolving this? In my opinion, I think the left Icon should be default to the component but we could use a prop like `simple` for a simple NavBar (no icons, just text). Right now we have `left` and `onLeftPress` which enables the user to do whatever he wants with the left part of the NavBar (You could just insert a <Block /> component there). But we need another way (easier and more intuitive). 

Thanks,
Alin 🤙🏽

**Comment by superandrew** [comment]
@palingheorghe I realized that since I was using https://github.com/creativetimofficial/material-kit-react-native their `Header` wasn't respecting the `left` property and that led me to think that the left view was something that was different than the left icon/menu button.

That said, I still think that a `simple` that allows a "no-icon" mode or even `menu` (true|false) property would help in applications that don't have a menu or similar left icon functionalities, or the section of the application.

**Comment by palingheorghe** [comment]
Yeah you're right @superandrew . This should be implemented, our component really needs that. You could submit a PR regarding this if you feel like contributing to Galio. We could update this in v0.6. 

Thanks,
Alin

**Comment by superandrew** [comment]
Sure, I will try to work out a solution. I linked galio trough `yarn link` but I have this error

    Unable to resolve "@babel/runtime/helpers/interopRequireWildcard" from "../galio/src/index.js"

Since I am not practical with local node modules development, am I missing something obvious or is this a setup mistake?

Thanks

**Comment by superandrew** [comment]
ouch I think that updating galio broke a whole lot of things since the release 1.5.3 changed a lot of things. Also, because I was using material kit from creativetim, I think most of the changes I made to the material kit layer are now incompatible.. so I'll have to rework it a little bit before making that change

**Comment by palingheorghe** [comment]
You could take a look at our change log [here](https://github.com/galio-org/galio/releases). I didn't change much besides the Icon component. This should help you make the process of updating your app faster.

**Comment by indper** [comment]
you can add:
left={(
        <Button
          color="transparent"
          style={styles.settings}
        >
        </Button>
      )}

**Comment by indper** [comment]
You can add
` left={(
        <Button
          color="transparent"
          style={styles.settings}
        >
        </Button>
      )}`

**Comment by shubhamkakkar** [comment]
I am working on this and will  soon raise a PR
